wgsl/index.bs

```diff
@@ -5634,36 +5637,30 @@ then:
 
       continuing {
         i = i + step;
-        if i >= 4 { break; }
+        if i >= 4 { break; } // Invalid.  Use break-if instead.
       }
     }
   </xmp>
 </div>
 
-<div class='example wgsl function-scope' heading="WGSL Valid loop if-else-break from a continuing clause">
-  <xmp>
-    var a: i32 = 2;
-    var i: i32 = 0;
-    loop {
-      let step: i32 = 1;
-
-      if i % 2 == 0 { continue; }
+### Break-If Statement ### {#break-if-statement}
 
-      a = a * 2;
+<div class='syntax' noexport='true'>
+  <dfn for=syntax>break_if_statement</dfn> :
 
-      continuing {
-        i = i + step;
-        if i < 4 {} else { break; }
-      }
-    }
-  </xmp>
+    | [=syntax/break=] [=syntax/if=] [=syntax/expression=] [=syntax/semicolon=]
 </div>
 
-<div class='example wgsl function-scope expect-error' heading="WGSL Invalid breaks from a continuing clause">
+A <dfn noexport dfn-for="statement">break-if</dfn> statement evaluates a boolean condition, and if the condition
+is true, transfers control to the first statement after the body of the nearest-enclosing [=statement/loop=] statement.
```

Comment:
Nit: this seems a bit wrong for a loop nested in a continuing block. Also an existing problem for regular breaks.
```
loop {
  if (c1) { break; }
  continuing {
    loop {
      continuing {
        break if c2;
      }
    }
  }
}
let x = 5; // This is the first statement after the inner loop
```

Reply:
Fixed it here and for a regular break, which had the same problem.